packages/eas-cli/src/credentials/ios/api/graphql/mutations/IosAppBuildCredentialsMutation.ts

```diff
@@ -113,7 +116,7 @@ const IosAppBuildCredentialsMutation = {
         )
         .toPromise()
     );
-    return data.iosAppBuildCredentials.setProvisioningProfile;
+    return data.iosAppBuildCredentials.setProvisioningProfile!;
```

Comment:
Hmm, if `!` is needed because the type can be nullable then maybe it'd be better to use `assert(...setProvisioningProfile)`/`nullthrows(...setProvisioningProfile)` instead of the type enforcement?

Reply:
The same comment applies to other places where you use `!`.